pytest_fixtures/oscap_fixtures.py

```diff
@@ -61,6 +61,6 @@ def tailoring_file(module_org, module_location, tailoring_file_path):
     ).create()
     return {
         "name": tailoring_file_name,
-        "tailoring_file_id": tf_info['id'],
+        "tailoring_file_id": tf_info.id,
```

Comment:
This was causing a TypeError because `tf_info` is an entity instance, not a dictionary.